src/StatusSummary.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+const Table = styled.table`
+  border-collapse: collapse;
+  text-align: center;
+  font-family: 'Gill Sans', sans-serif;
+`
+
+const CellHeader = styled.td`
+  color: NavajoWhite;
+  border: none;
+  font-size: 26px;
+  padding: 1px 10px 1px 10px;
+  letter-spacing: 1px;
+`
+
+const CellGreen = styled.td`
+  border: 3px solid #666666;
+  padding: 10px 10px 10px 10px;
+  font-size: 22px;
+  background: mediumseagreen;
+`
+
+const CellYellow = styled(CellGreen)`
+  background: goldenrod;
+`
+
+const CellRed = styled(CellGreen)`
+  background: indianred;
+`
+
+const StatusSummary = ({data}) => {
+  let vacant = 0
+  let ocupied = 0
```

Comment:
spelling: `ocupied` -> `occupied`

Reply:
Notice the spelling mistakes in a few places. Can do a global search and replace all `ocupied` to `occupied`